wordCreator.py

```diff
@@ -0,0 +1,294 @@
+import numpy as np
+import random
+
+class WordCreator(object):
+    def __init__(self, filename=None, words=None):
+        """
```

Comment:
Your method will probably break if an user does not provide `filename` and `words`. By getting rid of a default value, you can require the user to input the necessary information. My suggestion will be changing this line to be something like this:
```
def __init__(self, filename, words):
```